.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: required
+services: docker
+script: docker build . -t cs50/mysql:${TRAVIS_BRANCH/master/latest}
```

Comment:
What's `${TRAVIS_BRANCH/master/latest}` do here exactly?

Reply:
Set tag name to `latest` if git branch is `master`.

Reply:
neat